File: WebsiteTav/Models/Regular/TblHelp.cs
namespace WebsiteTav.Models.Regular
{
    public class TblHelp
    {
        public int id { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string Description { get; set; }

        public TblHelp(int id)
        {
            this.id = id;
        }

		public TblHelp(int id, string name, string email, string description)
        {
            this.id = id;
            Name = name;
            Email = email;
            Description = description;
        }
        public TblHelp(string name, string email, string description)
        {
            Name = name;
            Email = email;
            Description = description;
        }

        public TblHelp()
        {
            
        }
    }
}

File: WebsiteTav/Scripts/Admin/TblHelp.js
function LoadInHelps()
{
    const table = document.getElementById('tblMain');

    //---> int id
    //---> string Name
    //---> string Email
    //---> string Description
    const helps = SelectAllHelps();

    for (const entry of helps)
    {
        table.innerHTML +=
            `
                <tr>
                    <td>${entry.id}</td>
                    <td>${entry.Name}</td>
                    <td>${entry.Email}</td>
                    <td>${entry.Description}</td>
                </tr>   
                
            `;
    }
}

LoadInHelps();